src/ReleaseHistory.md

```diff
@@ -1,9 +1,12 @@
 # SARIF Package Release History (SDK, Driver, Converters, and Multitool)
 
+## **v3.1.0-beta1** [Sdk](https://www.nuget.org/packages/Sarif.Sdk/3.1.0-beta1) | [Driver](https://www.nuget.org/packages/Sarif.Driver/3.1.0-beta1) | [Converters](https://www.nuget.org/packages/Sarif.Converters/3.1.0-beta1) | [Multitool](https://www.nuget.org/packages/Sarif.Multitool/3.1.0-beta1) | [Multitool Library](https://www.nuget.org/packages/Sarif.Multitool.Library/3.1.0-beta1)
+
+* DEPENDENCY BREAKING: SARIF.SDK now requires `System.Collections.Immutable` 1.5.0. [#2504](https://github.com/microsoft/sarif-sdk/pull/2533)
```

Comment:
Is there a compatibility issue between 1.5.0 and 5.0.0? If 5.0.0 was previously required, is this really a breaking change or are we actually loosening restrictions for the sarif sdk?

Reply:
we have other systems that uses sarif-sdk which is causing major breaking changes in other projects.

So, with that in mind, I want to reduce to the previous version that we were using.